src/main/java/core/basesyntax/figures/RightTriangle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.figures;
+
+public class RightTriangle extends Figure {
+    private int side;
+    private int height;
+
+    public RightTriangle(String color, int side, int height) {
+        super(color);
+        this.side = side;
+        this.height = height;
+    }
+
+    @Override
+    public double getArea() {
+        return (0.5 * side * height) * 10.0;
```

Comment:
why do we multiply by 10?

Reply:
I experimented, and it seems that I did not delete it in this place